test/source/tests/tests/compose.ts

```diff
@@ -339,6 +339,21 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
       expect(await composePage.read('@input-body')).to.include('Test Draft Reply (Do not delete, tests is using this draft)');
     }));
 
+    ava.test('key-mismatch[global:compatibility] - standalone - key mismatch loading', testWithSemaphoredGlobalBrowser('compatibility', async (t, browser) => {
+      const appendUrl = 'isReplyBox=___cu_true___&threadId=16b584ed95837510&skipClickPrompt=___cu_false___&ignoreDraft=___cu_false___' +
+        '&threadMsgId=16b584ed95837510';
+      const replyMismatchPage = await ComposePageRecipe.openReplyKeyMismatch(t, browser, 'compatibility', appendUrl);
+      await Util.sleep(3);
+      const emailsPreview = await replyMismatchPage.waitAny('@email-preview');
+      const recipients = await emailsPreview.$$('span');
+      expect(recipients.length).to.equal(1);
+      const recipientEmail = await (await recipients[0].getProperty('textContent')).jsonValue() as string;
+      expect(recipientEmail).to.equal('flowcrypt.compatibility@gmail.com');
+      const text = await replyMismatchPage.read('@input-body');
+      expect(text).to.include('I was not able to read your encrypted message because it was encrypted for a wrong key.');
+      expect(await replyMismatchPage.isElementPresent('@attachment')).to.be.true;
```

Comment:
Please also try to send the message out, to see that there's no error (through mock).

Reply:
Ok

Reply:
Thanks